components/script/dom/htmliframeelement.rs

```diff
@@ -223,6 +224,12 @@ impl HTMLIFrameElement {
             return;
         }
 
+        if mode == ProcessingMode::FirstTime {
```

Comment:
Spec link? Why was that change needed? Please write more details in the commit message.

Reply:
This has been addressed through a more descriptive commit message, and a comment with a link to the spec.